framerail/src/routes/[slug]/[...extra]/+server.ts

```diff
@@ -0,0 +1,69 @@
+import * as page from "$lib/server/page"
+
+export async function POST(event) {
+  let data = await event.request.formData()
+  let slug = event.params.slug
+
+  let extra = event.params.extra
+    ?.toLowerCase()
+    .split("/")
+    .filter((flag) => flag.length)
```

Comment:
There is already logic to parse wikidot-paths. Presently it's used as part of the `ViewService`, to do page parsing/presentation. It returns a list of flags with all the extra fields already parsed.

Reply:
Yes, but this bit is the web client making request to svelte server instead of to deepwell, and this flag is for the svelte server to tell if the web operation is edit, move or history.

Reply:
This is in a `+server.ts` file, couldn't it ask deepwell to process the route? Or other relevant data?

Reply:
It probably comes down to if framerail making requests to deepwell just for parsing these flags being economical or not.

Reply:
Yeah. One reason I have `ViewService` is so you pass all the data once and have it process everything and return the response, instead of going back and forth.

The main reason I'm interested in having the logic there is so that the parsing of options is consistent. There are some edge cases that you have to be careful with.